manifest.json

```diff
@@ -36,8 +36,13 @@
   },
   "signer": {
     "image": "ghcr.io/hyperledger/firefly-signer",
-    "tag": "v1.1.9",
-    "sha": "515271ea722e6bf9601524880e7280a1fc052453733338d6ed474f85d6618aa1"
+    "tag": "v1.1.10",
+    "sha": "a06502872afda77d2a8c11a38a59c1c7b0eccc9ca3e7845711a2e2d6b98d31d2"
+  },
+  "tezossigner": {
+    "image": "ecadlabs/signatory",
+    "tag": "v1.1.10",
+    "sha": "2a48567c22af3ad9eb5860c1e29b1dea8aa73a7889e0e779fdfd7b882ae1458a"
```

Comment:
I don't think Signator needs to go in the `manifest.json`. Currently, the `manifest.json` is used for the following two reasons:

- Lists the images/tags of FireFly services (within the Hyplerledger project) that should be tested together
- Lists the images/tags of the containers used to compile FireFly core and its smart contracts

Currently, we do not list other third party services that may be required in some configurations. For example, we don't list specific versions of Besu, Geth, Fabric, etc. We also don't list a specific Postgres image, even though it is a requirement if you're not using SQLite. I tend to think of Signatory as falling in the same bucket. It's a runtime that is outside the project, but may be required as a runtime dependency in certain configurations.

In reality, FireFly is probably compatible with a wide range of versions of these services and we want to allow end users to upgrade FireFly independently of third-party services. The idea behind the `manifest.json` is if you are going to upgrade _FireFly_ here is the list of images that you should upgrade. How and when you upgrade third party services is up to you.